src/cf-expandables/usage.md

```diff
@@ -78,27 +78,29 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Expanded
 
-Sometimes you may want the expandable to be open by default.
-This is as easy as adding the `.o-expandable_content__onload-open` modifier to the `.o-expandable_content` block.
+Sometimes you may want the expandable to be open by default. This is as easy
```

Comment:
Yes, and I disagree with it because more often than not, it seems there is no natural break so then you're mixing natural breaks with 80 char breaks. This is a perfect example, the second sentence has no natural break. I'll return the first sentence to it's previous break and get this merged.